src/NodeVisitor/OptimizerNodeVisitor.php

```diff
@@ -125,7 +125,7 @@ private function optimizeRawFilter(Node $node, Environment $env): Node
     /**
      * Optimizes "for" tag by removing the "loop" variable creation whenever possible.
      */
-    private function enterOptimizeFor(Node $node, Environment $env): void
```

Comment:
The `$env` parameter have never been used since this method was created https://github.com/twigphp/Twig/commit/bd5629b3f24d47463691afa22a6da5cad0b1a108